go/common/crypto/address/address.go

```diff
@@ -0,0 +1,80 @@
+// Package address implements a generic cryptographic address.
+package address
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+)
+
+const (
```

Comment:
If there's only one item, don't need to make a block for it, `const Size = 20` will do.

Reply:
Yea, thanks!